test/e2e/alertmanager_instance_namespaces_test.go

```diff
@@ -27,7 +27,8 @@ import (
 )
 
 func testAlertmanagerInstanceNamespacesAllNs(t *testing.T) {
-	ctx := framework.NewTestCtx(t)
+	testCtx := framework.NewTestCtx(t)
```

Comment:
This variable is mostly unused, how about we make `NewTestCtx` return a pointer?